index.php

```diff
@@ -114,6 +114,15 @@
 
 // Set HTTP-Headers
 header('Content-Type: text/html; charset=utf-8');
+header('X-Frame-Options: sameorigin');
+header('X-XSS-Protection: 1; mode=block');
```

Comment:
I am not sure if we should use this header. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection

> Non-standard: This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.

and

> Warning: Even though this feature can protect users of older web browsers that don't yet support [CSP](https://developer.mozilla.org/en-US/docs/Glossary/CSP), in some cases, XSS protection can create XSS vulnerabilities in otherwise safe websites. See the section below for more information.

Reply:
```suggestion
```